deployments/engine/docker-compose/3m3e_with_s3.yaml

```diff
@@ -0,0 +1,194 @@
+version: "2.3"
+services:
+  server-master-0:
```

Comment:
Can add `restart` policy to server-master and executor containers.

Reply:
For normal test I prefer we don't restart on failure, in case a panic bug is ignored by restart.